Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/Artifact/Factory/TestImpactTestTargetMetaMapFactoryUtils.cpp

```diff
@@ -16,7 +16,7 @@ namespace TestImpact
         SuiteLabelSet labelSet;
         for (const auto& label : suiteLabels)
         {
-            const auto labelString = label.GetString();
+            const auto labelString = label.GetString(); 
```

Comment:
Extra whitetspace has been introduced on this line.
```suggestion
            const auto labelString = label.GetString();
```